mne/io/constants.py

```diff
@@ -230,6 +230,14 @@ def __setattr__(self, attr, val):
 FIFF.FIFF_SUBJ_COMMENT      = 409  # Comment about the subject
 FIFF.FIFF_SUBJ_HIS_ID       = 410  # ID used in the Hospital Information System
 
+FIFF.FIFFV_SUBJ_HAND_UNKNOWN= 0    # Unknown or mixed
```

Comment:
Is it in the Elekta FIFF definitions? We don't want to make a new one up.
If you don't know if you might just not assign a value.